cmd/run.go

```diff
@@ -38,6 +43,16 @@ func Run(configFile string) (out Stoppable, err error) {
 		return nil, fmt.Errorf("Error while loading config file: %v", err)
 	}
 
+	if c.PprofPort > 0 {
+		go func() {
+			log.Printf("starting pprof listener on port %d", c.PprofPort)
+			err := http.ListenAndServe(fmt.Sprintf(":%d", c.PprofPort), nil)
+			if err != nil {
```

Comment:
http.ListenAndServe will only return if `err` is non-nil
So the `if` is unnecessary.